_layouts/established.html

```diff
@@ -0,0 +1,118 @@
+---
+layout: default
+---
+
+<!-- Pull only the promos that exist for the specific location if any -->
+{% assign location_happenings = site.promos | where: "target_audience", page.name | sort: "date" | reverse %}
+{% assign series = site.series | last %}
+{% include header.html %}
+
+<!-- Resi Player -->
+{% assign locations_with_live_stream =  "oakley, florence, mason, eastside, westside, lexington, uptown" | split: ", "%}
```

Comment:
Let's move Resi into a partial so we do not need to maintain 2 sets of live stream locations